doc/integrator/fulltext_search.rst

```diff
@@ -32,11 +36,18 @@ Here is an example of an insertion in the ``tsearch`` table:
        to_tsvector('french', 'text to search', ' ', 'g'))
     );
 
-Where ``Layer group`` is the name of the layer group that should be activated,
-``text to display`` is the text that is displayed in the results,
```

Comment:
We lost this sentence, is it wanted?

Reply:
no, it was a mistake... I  update the PR.